### DISCUSSION TITLE
allow per-note circle size kinda like what they did to slider velocity

### DISCUSSION THREAD

**Discussion by prismotizm** [ppy/osu#18312] [Ideas] [answered]
i just think it would allow for some really really fun mapping and playing
![image](https://user-images.githubusercontent.com/60050794/168756133-d787de7f-4436-4f74-be4d-b77d2137a194.png)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13028

**Reply by prismotizm** [reply]
YESSSS
